package.json

```diff
@@ -25,7 +25,7 @@
     "@fortawesome/vue-fontawesome": "^0.1.6",
     "autosize": "^4.0.2",
     "axios": "^0.19.0",
-    "bootstrap-vue": "^2.0.0-rc.15",
+    "bootstrap-vue": "^2.0.0-rc.25",
```

Comment:
RC? are we using some release candidate version?

Reply:
it looks like so..

Reply:
I think they just don't want to release a non-rc version or something :D 
v2.0.0-rc.0
Released: 2018-01-23